src/App.js

```diff
@@ -4,94 +4,182 @@ import styled from 'styled-components';
 import { confluenceLinks } from './constants';
 
 const StyledApp = styled.div`
-  text-align: center;
-`;
-
-const StyledParagraph = styled.p`
-  color: red;
+  text-align: left;
 `;
 
 const StyledHeader = styled.header`
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
+  color: black;
+  background-color: #fff;
+  min-height: 10vh;
+  margin-left: 20px;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 `;
 
-const StyledLink = styled.a`
-  color: #61dafb;
+const StyledTitle = styled.p`
+  font-size: 28px;
 `;
 
-const random = number => {
-  return Math.floor(Math.random() * (number + 1));
-};
-
-const backgroundChange = () => {
-  return `rgb(${random(255)}, ${random(255)}, ${random(255)})`;
-};
+const StyledSubTitle = styled.p`
+  font-size: 12px;
+  color: #5e6c84;
+`;
 
-const changeStyling = () => {
-  console.log('The button has been clicked and the width and font family has been changed');
-  document.body.style.width = '30%';
-  document.body.style.backgroundColor = backgroundChange();
-  document.body.style.fontFamily = 'monospace';
-};
+const StyledGrid = styled.div`
+  max-width: 896px;
+  border: 1px solid #c1c7d0;
```

Comment:
We can create the table styling by adding:

```
  display: flex;
  flex-wrap: wrap;
```
This is because the grid is the parent component to the children within it. The way CSS works is they can inherit styles from their parent. 

`flex-flow: wrap` means that flex items will wrap onto multiple lines, from top to bottom. We can set how many items to allow per line too. We can do this by creating another styled component which will be the container for every table cell. 

Something like:
```
const StyledCell = styled.div`
  flex: 1 0 21%;
`;
```
`flex: 1 0 21%;` restricts the items per row to 4.